ddtrace/contrib/asyncio/patch.py

```diff
@@ -2,7 +2,9 @@
 
 from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
 
-from .helpers import _wrapped_create_task
+from . import context_provider
```

Comment:
I would advise against this, you have a circular reference here.

You should move the definition of `context_provider` to another file, `helpers.py` might make sense (?), and then just import/export `context_provider` in `__init__.py`